src/ParamPassing.cc

```diff
@@ -148,6 +184,13 @@ ElementPtr getElementById(const SDFPtr _sdf,
       childElem = matchingChild->GetFirstElement();
       matchingChild = nullptr;
       startIdx = longestIdx;
+
+      size_t found = _elemId.substr(startIdx).find("::");
+      if (found != std::string::npos && found == 1u)
+      {
+        // past double colons
+        startIdx += 3;
```

Comment:
Should this be 2 (for the number of characters in "::")?

Reply:
It is 3 because the `startIdx` points to the last character before `::` and needs to move to the character after `::`. For example in `link::visual`, before entering the if statement `startIdx=3` which points to character `k` and after incrementing by 3 it will point to `v`.